src/native/eventpipe/ep-event-source.c

```diff
@@ -38,6 +38,8 @@ const ep_char8_t* _ep_arch_info = "x64";
 const ep_char8_t* _ep_arch_info = "arm32";
 #elif defined(TARGET_ARM64)
 const ep_char8_t* _ep_arch_info = "arm64";
+#elif defined(TARGET_S390X)
+const ep_char8_t* _ep_arch_info = "s390x";
```

Comment:
Do you have a matching _ep_os_info for this CPU arch or do you need to add something new there as well?

Reply:
The OS is Linux, so that should already be handled by the existing code.